graphql/queryResolvers/getLessonMentors.ts

```diff
@@ -0,0 +1,24 @@
+import { User, UserLesson } from '../../helpers/dbload'
+
+type ArgsGetLessonMentors = {
+  lessonId: string
+}
+
+export const getLessonMentors = async (
+  _parent: void,
+  args: ArgsGetLessonMentors
+) => {
+  const { lessonId } = args
+  try {
+    const results = await UserLesson.findAll({
+      where: { lessonId },
+      include: [{ model: User }]
+    })
+
+    return results.map((result: any) => {
```

Comment:
Please fill in the type. We'd like to avoid using `any` since it defeats the purpose of TypeScript

Reply:
Got it. :smile: Will make that change.

Reply:
@hwong0305 What type should result be?